utopia-remix/app/env.server.ts

```diff
@@ -28,9 +32,13 @@ export const BrowserEnvironment: BrowserEnvironment = {
   EDITOR_URL: mustEnv('REACT_APP_EDITOR_URL'),
 }
 
-function mustEnv(key: string): string {
+function mustEnv(key: string, envFallback: Partial<Record<AppEnv, string>> = {}): string {
```

Comment:
you can use `optionalEnv` instead of `mustEnv`

Reply:
it's not optional for proper function, though (just uneeded for tests)

Reply:
I would definitely not put it there though imho, a `must` means it _must_ be set, you can set the related variable in the environment when running the tests

Reply:
+1 we shouldn't be giving an inline fallback to `mustEnv` like this. We should either use `mustEnv` as is and provide the test value when running the tests, or use `optionalEnv` to provide the fallback inline. The danger of doing it like this is that it implicitly makes the thing optional.

Reply:
this is not resolved yet, right? 🤔